EnviroMonitorFrontend/package.json

```diff
@@ -7,8 +7,9 @@
     "test": "mocha --compilers js:babel-core/register tests/**/*.spec.js",
     "test:watch": "npm run test -- --watch",
     "coverage": "nyc --reporter text --reporter text-summary --reporter html npm run test",
-    "start": "webpack-dev-server",
-    "build": "webpack"
+    "start": "webpack-dev-server --content-base ./build/",
+    "build": "webpack",
+    "build:watch": "webpack --progress --colors --watch"
```

Comment:
@lechup - sorry, I didn't have much time yestarday, as well today. I think you can remove the build:watch, though, if all of the changes were reflected :).

About nyc - we can open a new ticket for it, so we won't have to handle it here ;).